bindings/python/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+CMAKE_MINIMUM_REQUIRED (VERSION 2.6)
+
+INCLUDE (${CMAKE_SOURCE_DIR}/cmake/python.cmake)
+
+PROJECT (state_observation_py)
+
+# Dependencies.
+FINDPYTHON ()
+ADD_REQUIRED_DEPENDENCY ("eigenpy >= 1.3.0")
```

Comment:
To avoid craches, you must use eigenpy>= 1.3.1. It fixes a serious bug due to the linkage procedure.